doc/articles/guides/localization-and-culture-hot-swapping.md

```diff
@@ -0,0 +1,186 @@
+# How to use localization and perform runtime culture switching
+## Full Sample
+Both complete sample code can be found here:
```

Comment:
Modify the intro to match the template: https://github.com/unoplatform/uno/blame/dev/djo/docs-fix-3/doc/.howto-template.md